utilities/_templates/react/src/default.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { forwardRef } from "react";
+import cl from 'classnames';
+import '@nav-fronted/<%name.indexfileUncapped%>-styles';
+
+export interface <%name.stripped%>Props {
+    /**
+     * User defined classname
```

Comment:
Skriver du dokumentasjon på norsk eller engelsk?

Reply:
Godt spørsmål, `User defined classname` er det som vil bli brukt av docgen og lagt på siden så burde kanskje være norsk?

Reply:
Jeg stemmer for engelsk i kommentarer, så slipper vi å blande språk i koden. Våre komponenter vil jo heller ikke handle om domene/lovverk, bare standard ui-patterns.